### DISCUSSION TITLE
Imagem dos produtos não carregam

### DISCUSSION THREAD

**Discussion by AleDinizRJ** [balta-io/7187#43] [Perguntas e Respostas] [answered]
Olá pessoal, boa tarde! 

A minha aplicação não está carregando as imagens dos produtos do PETSHOP. Veja na imagem abaixo:
![image](https://user-images.githubusercontent.com/88723782/134785297-2bf0eeab-5f08-441f-9bbc-aa8884af62af.png)

Eu executei a API pelo Mockoon e estou com a minha página HTLM da mesma forma como foi apresentado no curso. Veja abaixo:
![image](https://user-images.githubusercontent.com/88723782/134785500-b7cb37d7-42f4-416a-9477-c986aa2d2b51.png)

Por favor! Preciso de uma ajuda.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @AleDinizRJ, como vai? As imagens não estão mais disponíveis, elas não vão carregar, infelizmente...
Você pode usar algum placeholder para gerar esta imagem:
```html
<img src="https://picsum.photos/200/300" />
```

**Reply by marquesleo** [reply]
como faço isso balta ?

**Reply by andrebaltieri** [reply]
Bom dia @marquesleo posta um trecho do seu Card ai (HTML) que te ajudo melhor <3
